dataflow/src/org/checkerframework/dataflow/analysis/AnalysisResult.java

```diff
@@ -31,6 +32,9 @@
     /** Map from AST {@link Tree}s to {@link Node}s. */
     protected final IdentityHashMap<Tree, Node> treeLookup;
 
+    /** Map from AST {@link UnaryTree}s to compound {@link AssignmentNode}s. */
+    protected final IdentityHashMap<UnaryTree, AssignmentNode> compoundAssignTreeLookupMap;
```

Comment:
`compoundAssignTreeLookupMap` seems like a bad name for this map.
A compound assignment is something like `a += b`, but this map stores a UnaryTree to AssignmentNode mapping. Is the assignment node the artificially introduced one for the pre/post inc/dec? In any event, there is no compound assignment involved, is there?